packages/webapp/src/components/Modals/WaterUseCalculatorModal/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import ModalComponent from '../ModalComponent/v2';
+import { ReactComponent as Person } from '../../../assets/images/task/Person.svg';
+import styles from '../QuickAssignModal/styles.module.scss';
+import Button from '../../Form/Button';
+
+export interface IWaterUseCalculatorModal {
+  dismissModal: () => void;
+}
+
+const WaterUseCalculatorModal: FC<IWaterUseCalculatorModal> = ({ dismissModal }) => {
+  const { t } = useTranslation();
+  return (
+    <ModalComponent>
```

Comment:
@wasswarichard I think this tag is meant to be self-closing, the attributes are showing up as text in the modal currently.